hack/validate/vendor

```diff
@@ -3,28 +3,49 @@
 export SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPTDIR}/.validate"
 
-IFS=$'\n'
-files=( $(validate_diff --diff-filter=ACMR --name-only -- 'vendor.conf' 'vendor/' || true) ) 
-unset IFS
+validate_vendor_diff(){
+	IFS=$'\n'
+	files=( $(validate_diff --diff-filter=ACMR --name-only -- 'vendor.conf' 'vendor/' || true) )
+	unset IFS
 
-if [ ${#files[@]} -gt 0 ]; then
-	# We run vndr to and see if we have a diff afterwards
-	vndr
-	# Let see if the working directory is clean
-	diffs="$(git status --porcelain -- vendor 2>/dev/null)"
-	if [ "$diffs" ]; then
-		{
-			echo 'The result of vndr differs'
-			echo
-			echo "$diffs"
-			echo
-			echo 'Please vendor your package with github.com/LK4D4/vndr.'
-			echo
-		} >&2
-		false
+	if [ ${#files[@]} -gt 0 ]; then
+		# We run vndr to and see if we have a diff afterwards
+		vndr
+		# Let see if the working directory is clean
+		diffs="$(git status --porcelain -- vendor 2>/dev/null)"
+		if [ "$diffs" ]; then
+			{
+				echo 'The result of vndr differs'
+				echo
+				echo "$diffs"
+				echo
+				echo 'Please vendor your package with github.com/LK4D4/vndr.'
+				echo
+			} >&2
+			false
+		else
+			echo 'Congratulations! All vendoring changes are done the right way.'
+		fi
 	else
-		echo 'Congratulations! All vendoring changes are done the right way.'
```

Comment:
github diff is weird, but there is no code change for the existing code (but there is space->tab consistency fix)